hw/ip/i2c/doc/_index.md

```diff
@@ -142,6 +143,30 @@ Typically every byte transmitted must also receive an ACK signal, and the IP wil
 However, there are some I2C commands which do not require an ACK.
 In those cases this flag should be asserted with FBYTE indicating no ACK is expected and no interrupt should be raised if the ACK is not received. 
 
+### Target Address Registers
+
+I2C target device is assigned two 7-bit address and 7-bit mask pairs.
+The target device accepts a transaction if the result of the bitwise AND operation performed on the transaction address sent by the host and a mask matches the assigned address corresponding to the mask.
+Thus, the target device may respond to any of the two assigned unique addresses.
+The assigned address and mask pairs are set in registers {{< regref ADDRESS0 >}}, {{< regref MASK0 >}}, {{< regref ADDRESS1 >}}, and {{< regref MASK1 >}}. 
+
+### Acquired Formatted Data
+
+This section applies to I2C in the target mode.
+When the target accepts a transaction, it inserts the transaction address, read/write bit, and START signal sent by the host into the ACQ FIFO where they can be accessed by software.
+ACQ FIFO output corresponds to {{< regref ACQDATA >}}.
+If the transaction is a write operation (R/W bit = 0), the target proceeds to read bytes from the bus and insert them into the ACQ FIFO until the host terminates the transaction by sending a STOP or RESTART signal.
+A STOP or RESTART indicator is inserted into the ACQ FIFO as the next entry following the last byte read, in which case other bits are junk.
+If the transaction is a read operation (R/W bit = 1), the target pulls bytes out of TX FIFO and transmits them to the bus until the host signals the end of the transfer by sending a NACK signal.
+TX FIFO input corresponds to {{< regref TXDATA >}}.
+Typically, a NACK signal is followed by a STOP or RESTART signal and the IP will raise an exception if the host sends a STOP signal after an ACK.
+An ACK/NACK signal is inserted into the ACQ FIFO as the first bit (bit 0), in the same entry with a STOP or RESTART signal. 
+For ACK and NACK signals, the value of the first bit is 0 and 1, respectively.
+The ACQ FIFO entry consists of 10 bits:
+- Address (bits 0-6) and R/W bit (bit 7) or data byte
```

Comment:
If you are using the same reassembly as for bytes then won't the address be in bits 7:1 of the register and the R/W in bit 0 since the most significant bit is the first sent on the wire?

Reply:
You're right, the same reassembly is used.